operator/charts/paladin-operator/values.yaml

```diff
@@ -1,8 +1,8 @@
-
-
-# Install CRDs as part of the chart installation
-# When this is set to false, the CRDs are expected to be installed separately
-installCRDs: false
+# Installation mode. This setting determines which Custom Resources (CRs) will be installed by default when deploying this chart.
+# Supported modes:
+# - dev-net: Installs a default Paladin network (3 nodes) along with the related Smart Contracts.
+# - none (or left empty): Only the operator will be installed.
+mode: dev-net
```

Comment:
nit: `devnet` I think. Matches how blockchain communities refer to their broad networks as `testnet` and `mainnet` often in config.